src/pages/components/overflow-menu/style.mdx

```diff
@@ -10,23 +10,23 @@ tabs: ['Usage', 'Style', 'Code', 'Accessibility']
 
 | Class                                        | Property         | Color token                      |
 | -------------------------------------------- | ---------------- | -------------------------------- |
-| `.bx--overflow-menu__icon`                   | fill             | `$icon-01`                       |
-| `.bx--overflow-menu-options`                 | background-color | `$ui-01`                         |
-| `.bx--overflow-menu-options__btn`            | color            | `$text-02`                       |
-| `.bx--overflow-menu-options__option--danger` | background-color | `$support-01`                    |
+| `.bx--overflow-menu__icon`                   | fill             | `$icon-primary`                  |
+| `.bx--overflow-menu-options`                 | background-color | `$layer`                         |
+| `.bx--overflow-menu-options__btn`            | color            | `$text-secondary`                |
+| `.bx--overflow-menu-options__option--danger` | background-color | `$support-error`                 |
```

Comment:
This is an bug doc reference, should have been `ui-01` as well here. So change to `$layer`